spec/classes/mod/ldap_spec.rb

```diff
@@ -28,11 +28,11 @@
           ldap_trusted_global_cert_file: 'ca.pem',
           ldap_trusted_global_cert_type: 'CA_DER',
           ldap_trusted_mode: 'TLS',
-          ldap_shared_cache_size: '500000',
-          ldap_cache_entries: '1024',
-          ldap_cache_ttl: '600',
-          ldap_opcache_entries: '1024',
-          ldap_opcache_ttl: '600',
+          ldap_shared_cache_size: 500_000,
```

Comment:
Is the underscore a typo?

Reply:
No, in Ruby this is a convention to make numbers easier to read. Because it's quite hard to see if it's 5 or 6 zeros, this makes it visually easier to see it's 500k, not 5m. Rubocop even enforces it.

Reply:
Amazing, never knew this 👍 will definitely come in handy at some point :-)

Reply:
@pmcmaw: Sharp Eyes 👀  
I do not like it, but Rubocop with its Config-File coming from the PDK-Templates sais so.